### PR TITLE
DOCSP-11167: [VSCODE] Display indexes and index keys in tree view

### PR DESCRIPTION
**Pull request by steveren**
[Jira](https://jira.mongodb.org/browse/DOCSP-11167)
[Staged](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-vscode/docsworker-xlarge/DOCSP-11167/databases-collections)

### CODE REVIEW

File: source/databases-collections.txt [hunk 1]
```diff
@@ -58,7 +58,9 @@ Alternatively, right-click a collection and click
 in an array.
 
 Opening collection documents provides a **read-only** view of your data.
-To modify your data using |vsce|, use a JavaScript Playground.
+To modify your data using |vsce|, use a :ref:`JavaScript Playground
+<vsce-crud>` or launch a shell by right-clicking your active deployment
+in the left-side navigation.
```
Comment (steveren) [inline source/databases-collections.txt:63]: Off-ticket improvement here.

File: source/databases-collections.txt [hunk 2]
```diff
@@ -75,6 +77,24 @@ appear in that collection's documents. If a field exists in all
...
+Your collections's indexes appear under the :guilabel:`Indexes` heading.
+When you expand an index, each index key appears with an icon designating
+its type. Index key types include:
+
+- Ascending
+- Descending
+- Geospatial (2d, 2dsphere, geoHaystack)
+- Text
+- Hashed
+
```
Comment (steveren) [inline source/databases-collections.txt:92]: Note: Atlas Search indexes aren't listed. I put a comment in the ticket to inquire.

File: source/databases-collections.txt (hunk 1 above)
Comment (jwilliams-mongo) [reply source/databases-collections.txt:63]: nit: the navigation might not be on the left side -- I moved it to the right side. Consider: ... in the MongoDB view in the Activity Bar. Looks like this is what we use [elsewhere](https://docs.mongodb.com/mongodb-vscode/connect#disconnect-from-a-deployment).

Comment (steveren) [reply source/databases-collections.txt:63]: Done.

**Pull request merged**